config/webhook/manifests.yaml

```diff
@@ -201,6 +201,5 @@ webhooks:
     - v1alpha3
     operations:
     - UPDATE
-    - DELETE
```

Comment:
shall we add CREATE ?

Reply:
There are no validations during creation.